### DISCUSSION TITLE
Verifying idToken in Backend

### DISCUSSION THREAD

**Discussion by gaoyue945** [Web3Auth/Web3Auth#1192] [Help]
Whether jwt is encrypted

What is the encryption key if it is encrypted

**Comment by shahbaz17** [comment]
Hey @gaoyue945 

JWTs are not encrypted. You can verify the idToken at your backend by following the steps mentioned https://web3auth.io/docs/server-side-verification/social-login-users#verifying-jwt-token-idtoken

**Reply by gaoyue945** [reply]
ok,thanks
